azurerm/resource_arm_container_group_test.go

```diff
@@ -595,8 +766,7 @@ func testCheckAzureRMContainerGroupExists(name string) resource.TestCheckFunc {
 			}
 			return fmt.Errorf("Bad: Get on containerGroupsClient: %+v", err)
 		}
-
-		return nil
+		return checkResp(resp)
```

Comment:
We should only be checking the resource exists here, anything more should be via the state.